config/application.rb

```diff
@@ -127,5 +127,7 @@ class Application < Rails::Application
     config.x.recording_token_ttl = ENV.fetch('PROTECTED_RECORDINGS_TOKEN_TIMEOUT', '60').to_i.minutes
     # Protected recordings resource access cookie timeout in minutes. Defaults to 360 (6 hours)
     config.x.recording_cookie_ttl = ENV.fetch('PROTECTED_RECORDINGS_TIMEOUT', '360').to_i.minutes
+
+    config.i18n.default_locale = ENV.fetch('LOCALE', 'en')
```

Comment:
This variable should probably be named "DEFAULT_LOCALE" to better reflect what it does.

If multiple locales are available in the locale file, and the user's browser is set to some other locale that has strings available, then the other locale will be used instead.